src/pluto/grey_image.cr

```diff
@@ -0,0 +1,63 @@
+require "./image"
+
+class Pluto::GreyImage < Pluto::Image
+  property grey : Array(UInt8)
+  property width : Int32
+  property height : Int32
+
+  def self.new(red : Array(UInt8), green : Array(UInt8), blue : Array(UInt8), alpha : Array(UInt8), width : Int32, height : Int32)
```

Comment:
I think it makes sense to remove this method since we already have `RGBImage#to_grey`?

Reply:
@Vici37